codegen/.rubocop.yml

```diff
@@ -18,6 +18,9 @@ Style/Documentation:
   Exclude:
     - '**/*_spec.rb'
 
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
Why are we changing this?

Reply:
Because rubySource is an inner write. The outside will have it.